resources/views/content/top-links.blade.php

```diff
@@ -1,41 +1,39 @@
-<x-section class="mb-3 flex flex-col gap-2">
-    <a class="btn text-center py-2" href="/globalRanking.php?s=5&t=2">
-        <span class="text-yellow-400"><x-fas-medal/></span>
+<x-section class="top-links">
+    <a href="/globalRanking.php?s=5&t=2">
+        <span class="text-yellow-500"><x-fas-medal/></span>
         Global Ranking
     </a>
-    <a class="btn text-center py-2" href="{{ route('download.index') }}">
-        <span class="text-heading"><x-fas-gamepad/></span>
+    <a href="{{ route('download.index') }}">
+        <span class="text-purple-400"><x-fas-gamepad/></span>
         Download Emulator
     </a>
     @if(config('services.discord.invite_id'))
-        <a class="btn text-center py-2" href="https://discord.gg/{{ config('services.discord.invite_id') }}">
+        <a href="https://discord.gg/{{ config('services.discord.invite_id') }}">
             <span class="text-discord"><x-fab-discord/></span>
             Join us on Discord
         </a>
     @endif
     @if(config('services.patreon.user_id'))
-        <a class="btn text-center py-2" href="https://www.patreon.com/bePatron?u={{ config('services.patreon.user_id') }}">
+        <a href="https://www.patreon.com/bePatron?u={{ config('services.patreon.user_id') }}">
             <span class="text-patreon"><x-fab-patreon/></span>
             Become a Patron
         </a>
     @endif
-    <div class="flex gap-1">
-        <a class="btn text-center py-2 grow" href="https://news.retroachievements.org/">
-            <span class="text-heading"><x-fas-newspaper/></span>
+    <div class="grid grid-cols-4">
+        <a href="https://news.retroachievements.org/">
+            <span><x-fas-newspaper/></span>
             RANews
         </a>
-        <a class="btn text-center py-2 grow" href="https://www.youtube.com/channel/UCIGdJGxrzmNYMaAGPsk2sIA">
-            <span class="text-heading"><x-fas-microphone/></span>
+        <a href="https://www.youtube.com/channel/UCIGdJGxrzmNYMaAGPsk2sIA">
```

Comment:
It has been suggested that this link should be updated to `https://www.youtube.com/@RAPodcast`: https://discord.com/channels/310192285306454017/490333108621672448/1080659700967813180

Reply:
I would be really grateful if we could go ahead with this change please, whenever is convenient

Reply:
PR to adjust the link: https://github.com/RetroAchievements/RAWeb/pull/1426